### ISSUE TITLE
Still in development?

### ISSUE THREAD

**Issue by petrk94** [ntoll/drogulus#38]
Is it still in development?

**Comment by ntoll** [comment]
Yes, currently being re-written (and simplified) to conform to https://sans-io.readthedocs.io/.

**Comment by petrk94** [comment]
@ntoll last commit in this repository branch is from 2016. Where I can follow the development?
